Userland/Utilities/chown.cpp

```diff
@@ -66,10 +80,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         }
     }
 
-    if (dont_follow_symlinks) {
-        TRY(Core::System::lchown(path, new_uid, new_gid));
-    } else {
-        TRY(Core::System::chown(path, new_uid, new_gid));
+    for (auto& p : paths) {
+        auto e = change_owner(p, new_uid, new_gid, dont_follow_symlinks);
+        if (e.is_error()) {
+            warnln("error while changing ownership of '{}'", p);
+            return e;
```

Comment:
Since we are returning here anyways, why not simply use `TRY()` as usual?

Reply:
I thought it would be nice to show what file failed to be `chown`ed

Reply:
This is a limitation of our error handling in general, so I'd personally like to see a solution that covers all tools instead of adding unique solutions to each tool separately. But that's a decision that's up to the maintainer who reviews this PR.